src/pull_request.rs

```diff
@@ -311,14 +315,14 @@ pub fn handle_pull_request(
 		reviews
 			.iter()
 			.find(|r| r.user.id == owner.id)
-			.map_or(false, |r| r.state == "APPROVED")
+			.map_or(false, |r| r.state == Some("APPROVED".to_owned()))
```

Comment:
A micro optimisation here would be to convert `state` into `Option<&str>` so that you're not allocating a string on every comparison. You can do this with `r.state.as_deref()`